plugins/source/azuredevops/codegen/recipes/templates/resource.go.tpl

```diff
@@ -0,0 +1,11 @@
+// Code generated by codegen; DO NOT EDIT.
+
+package {{.Service}}
+
+import (
+	"github.com/cloudquery/plugin-sdk/schema"
+)
+
+func {{.SubService | ToCamel}}() *schema.Table {
```

Comment:
if you want to supposr relations starting from lower case, take a look at https://github.com/cloudquery/cloudquery/blob/main/plugins/source/tailscale/codegen/resources/templates/resource.go.tpl#L10-L14

Reply:
Thanks for the reference. If we need to add relations we could use that